lihaofd/incubator-mxnet
# Support Quantilized Fully Connected

## Description ##
In this PR, it created quantilized fully connected op by using int8 gemm
@pengzhao-intel, @TaoLv , @ciyongch 

## Feature changes ##
### New features ###
- Support quantilized fully connected op by using int8 gemm
- Support int8 bias by using beta offset

### Unit-test changes ###
- Update testcase test_quantized_fc in tests/python/quantization/test_quantization.py
- Check consistency with original mx.sym.FullyConnected implementation.

## Checklist ##
